### DISCUSSION TITLE
Full world scan for unused chunks/regions.

### DISCUSSION THREAD

**Discussion by TomLewis** [Jikoo/Regionerator#61] [Q&A]
Hi
Brand new to Regionerator.

I have a paper 1.12.2 server with an 8 year old world ~285,000 players joined over these years.
I'm testing 1.13+ performance, and part of my testing will be to find a plugin to clean up unused chunks while keeping GriefPrevention and WorldGuard claims protected.

I had my eye on WorldBeautifier which seems to have vanished into thin air, but I found Regionerator as a replacement.
I don't have the time to wait for players to play and generate a new database, I need to find a way to scan the entire world on a test server and delete these chunks/regen them (And log them) as an entire world.

Here https://github.com/Jikoo/Regionerator/wiki/FAQ#why-is-it-taking-so-long-to-start-deleting-area you state you fall back to the internal last modification timestamp, is there any way to force scan for this? This is what was available in WorldBeautifier before it vanished. They had options for specifying how old to check too.

As long as WorldGuard and GriefPrevention are checked for in the chunk, then everything else that's not claimed and hasn't been used in a while is good to regen.

Can I do this with the plugins current version?

**Comment by Jikoo** [comment]
What you can do is set your visit duration, then [tell Regionerator that data has been gathering for a while](https://github.com/Jikoo/Regionerator/wiki/Testing#testing-deletion). Since there isn't any data for any chunks, Regionerator will fall through to their modification time. I do plan to decouple those modes eventually, but considering that I only just got around to per-world nodes in 3ed8b6ba5abedeab26b6452f1ca371da609b49b5 last month after it was requested half a year ago it'll probably be a while. I'll open an issue so people are aware it's on my radar.
